binder/views.py

```diff
@@ -67,6 +68,42 @@ def get_default_annotations(model):
 	return annotations
 
 
+def split_path(path):
+	"""
+	This function splits a dot seperated path into a list of keys.
```

Comment:
I still don't understand what this is doing, and why we need this function in the first place. This still seems like something which can be done in a regex.

Can you give some examples in the documentation?

Reply:
Even even better, can you add some unittests for this methods? Then we know for sure it is working as well